app/controllers/requests_controller.rb

```diff
@@ -9,10 +9,12 @@ def new; end
   def create
     question = params[:question]
     Request.create!(text: question)
-    QuestionMailer
-      .with(question: question)
-      .new_question_email
-      .deliver_later
+    User.where.not(email: nil).find_each do |user|
+      QuestionMailer
```

Comment:
I know this wasn’t introduced in this PR – but I was slightly confused why it’s called `QuestionMailer` as we don’t have questions any more, but requests. Probably something, we should think about when we have clear terms for everything.

Reply:
:+1: